frontend/src/_styles/theme.scss

```diff
@@ -2077,13 +2077,8 @@ input:focus-visible {
 
 input[type=text] {
   border-radius: 0 !important;
-  outline-color: #dadcde !important;
 }
 
-.widget-header {
```

Comment:
Need this class for the widget sidebar header

Reply:
I see, will update

Reply:
Have reverted the changes. Was from merge conflict resolution.